tests/unit/woopay/test-class-woopay-order-status-sync.php

```diff
@@ -6,17 +6,43 @@
  */
 
 use WCPay\WooPay\WooPay_Order_Status_Sync;
+use PHPUnit\Framework\MockObject\MockObject;
 
 /**
  * WooPay_Utilities unit tests.
  */
 class WooPay_Order_Status_Sync_Test extends WP_UnitTestCase {
 
 	/**
-	 * @var WP_User $admin_user
+	 * @var WP_User
 	 */
 	protected static $admin_user;
 
+	/**
+	 * @var WC_Payments_Account|MockObject
+	 */
+	private $account_mock;
+
+	/**
+	 * @var WC_Payments_API_Client|MockObject
+	 */
+	private $api_client_mock;
+
+	/**
+	 * @var WooPay_Order_Status_Sync|MockObject
+	 */
+	private $webhook_sync_mock;
+
+	/**
+	 * @var WCPay\Database_Cache $cache
```

Comment:
Nit: Since we removed the variable name (`$cache`) from line 17. Should we also remove it here?

Reply:
Nice catch. Fixed in [cbdaa6e](https://github.com/Automattic/woocommerce-payments/pull/8610/commits/cbdaa6e04cae96adea9394bee5b28709dec5a9ea).